blocks/test/fixtures/core__chat.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:core/chat {"compact":false} -->
+Pascal: Alexa, open the window
+Alexa: I'm not quite sure how to help you with
+Pascal: Siri, tell me a joke
+Siri: Pascal, get Siri-ous. Ha ha!
+Pascal: OK, Google. Now it's your turn!
+Google Home: Alexa, tell Siri to tell Pascal a better joke.
+Pascal: Ugh!
+Okay, that's enough. No internet for you anymore!
+Alexa: 💔
+Google Home: 😔
+Siri: 📴
```

Comment:
Yeah, that's a problem I still need to address, see https://github.com/WordPress/gutenberg/pull/1632#issuecomment-312481519.

You can edit the block, but you will see all the HTML markup in the textarea.

I would love to store only the raw value, but then how can it be rendered on the front end?

Feedback last week was that it should be stored with (as little as possible) markup in the database and then we'll see from there.